src/virtual-list/src/VirtualList.ts

```diff
@@ -386,9 +386,13 @@ export default defineComponent({
               return
             }
           }
-          e.preventDefault()
-          listEl.scrollTop += e.deltaY / ensureWheelScale()
-          listEl.scrollLeft += e.deltaX / ensureWheelScale()
+          e.preventDefault();
+          if (!e.shiftKey) {
```

Comment:
you can not only use shitKey to distinguish
shit+wheel works fine on MacOS
this case seems only happen on windows Chrome